subsys/bluetooth/services/fast_pair/fp_crypto/Kconfig.fp_crypto

```
@@ -37,6 +37,11 @@ config BT_FAST_PAIR_CRYPTO_MBEDTLS
 
 endchoice
 
+config BT_FAST_PAIR_CRYPTO_EXT_PN
+	bool "Fast Pair crypto for Personalized Name extension"
```

I would say that it is nice to keep prompt as short as possible. You can remove Fast Pair suffix for all prompt in this file. For example:

```suggestion
	bool Crypto for Personalized Name extension"
```